raiden/network/transport/udp/udp_transport.py

```diff
@@ -158,7 +160,8 @@ def __init__(self, discovery, udpsocket, throttle_policy, config):
         self.nat_keepalive_timeout = config['nat_keepalive_timeout']
         self.nat_invitation_timeout = config['nat_invitation_timeout']
 
-        self.event_stop = Event()
+        self.stop_event = Event()
+        self.stop_event.set()
```

Comment:
I believe this event should not be set here

Reply:
These events starts `set()`ed to tell whoever may care (this one not, but other classes checks for `stop_event` to see if the service is not running) that the object is initialized as stopped. It's cleared upon `start`, to indicate it's running. `stop_event` is then set again to wake up listening greenlets that the service is stopping.